src/main/resources/org/jenkinsci/plugins/azure/AzurePublisherSettings/credentials.jelly

```diff
@@ -27,57 +27,14 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
 
-  <j:set var="uniqueId" value="${h.generateId()}" />
-  <f:entry title="${%Publisher Settings}">
-    <input type="file" id="${uniqueId}-file"/>
-    <a href="http://go.microsoft.com/fwlink/?LinkId=254432" target="_blank">
-      Get it from microsoft.com
-    </a>
-  </f:entry>
-
-  <f:entry title="${%Subscription ID}" field="subscriptionId">
-    <f:textbox clazz="${uniqueId}-Id"/>
-  </f:entry>
-
-  <f:entry title="${%Subscription Name}" field="subscriptionName">
-    <f:textbox clazz="${uniqueId}-Name"/>
-  </f:entry>
-
-  <f:entry title="${%Service Management URL}" field="serviceManagementUrl">
-    <f:textbox clazz="${uniqueId}-ServiceManagementUrl" default="https://management.core.windows.net"/>
-  </f:entry>
-
-  <f:entry title="${%Service Management Certificate}" field="serviceManagementCert">
-    <f:textarea class="${uniqueId}-ManagementCertificate"/>
-  </f:entry>
+  <f:block>
+    <label class="error">
+      The Jenkins Azure Publisher Settings Plugin is deprecated with the deprecation of Azure classic deployments and
+      their replacement by ARM deployments. Please see recent Jenkins Azure plugins for alternatives (Azure App Service,
+      Azure Functions, Azure Container Service...) at <a href="https://plugins.jenkins.io/">https://plugins.jenkins.io/</a>.
+    </label>
+  </f:block>
 
   <st:include page="id-and-description" class="${descriptor.clazz}"/>
```

Comment:
Any reason to keep this include ?

Reply:
Only because I found the error I added above wasn't visible unless this include was present, and tbh I don't understand why that is with my very limited Jelly knowledge!
Happy to remove it if anyone has a pointer, but given the nature of the change I ended up leaving it in rather than wasting more time on it.